ansible/group_vars/proxy-hosts.yml

```diff
@@ -346,13 +357,12 @@ idr_haproxy_frontend_omero_host: idr.openmicroscopy.org
 ######################################################################
 # Static webpages (/about) on proxy
 
-idr_openmicroscopy_org_version: 2019.06.17
+idr_openmicroscopy_org_version: 923146aa2d77a49ee61f8330536265aaf6f4dd15
 deploy_archive_dest_dir: /srv/www/{{ idr_openmicroscopy_org_version }}
-deploy_archive_src_url: https://github.com/IDR/idr.openmicroscopy.org/releases/download/{{ idr_openmicroscopy_org_version }}/idr.openmicroscopy.org.tar.gz
+deploy_archive_src_url: https://a.uguu.se/4DOqxok8Gi3q_idr.openmicroscopy.org.tar.gz
```

Comment:
Note this is a temporary time limited (24h) upload, obviously it should be reverted when the website is tagged